openshift/patroni-postgis/docker/contrib/root/usr/share/scripts/patroni/disk_space_check.sh

```diff
@@ -35,7 +35,7 @@ send_notification() {
         -H 'X-Auth-Token: ${ROCKET_AUTH_TOKEN}' \
         -H 'X-User-Id: ${ROCKET_USER_ID}' \
         -H 'Content-Type: application/json' \
-        https://chat.pathfinder.gov.bc.ca/api/v1/chat.postMessage \
+        https://chat.developer.gov.bc.ca/api/v1/chat.postMessage \
```

Comment:
Keep in mind we have to re-build our patroni image, and then re-deploy the pods in production for this to take effect. (I've updated the ticket appropriately - https://app.zenhub.com/workspaces/wildfire-predictive-services-5e321393e038fba5bbe203b8/issues/bcgov/wps/657)

Reply:
Oops. Didn't realize that we already had a ticket for this. Gonna undo this change then, might as well leave it to be taken care of in the actual ticket.

Reply:
No - it's fine - I've already marked this work as done on that ticket. It's great - please keep it.